java/merchant-client/src/main/java/no/bankaxept/epayment/client/merchant/SimulationRefundRequest.java

```diff
@@ -0,0 +1,28 @@
+package no.bankaxept.epayment.client.merchant;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import no.bankaxept.epayment.client.base.SimulationRequest;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@JsonIgnoreProperties(value = {"simulationValues"})
```

Comment:
Would `@JsonIgnoreType` do the same?

Reply:
Is the point to not show/leak/something the simulationValues? I might be wrong, but I thought `@JsonIgnoreType` would do that with the whole class. Not sure if that is beneficial or just the same. I was thinking that this class could be included as an empty map in some cases. But I am not really sure of the use case here.

Reply:
This makes sure `simulationValues` is not marshalled but all the other properties including those on the super class. Which is what we want, as the client converts simulation values into an HTTP header.